packages/paste-core/components/combobox/src/styles/ComboboxListboxOption.tsx

```diff
@@ -39,14 +39,20 @@ const ComboboxListboxOption = React.forwardRef<HTMLLIElement, ComboboxListboxOpt
       <Box
         {...safelySpreadBoxProps(props)}
         as="li"
+        element={`${element}_LIST_ITEM`}
         backgroundColor={highlighted ? 'colorBackgroundPrimaryWeakest' : 'colorBackgroundBody'}
         padding="space30"
         cursor="pointer"
         ref={ref}
         {...virtualItemStyles}
         {...VariantStyles[variant]}
       >
-        <Text as="span" color="colorText" textDecoration={highlighted ? 'underline' : null}>
+        <Text
+          as="span"
+          color="colorText"
+          element={`${element}_LIST_ITEM_TEXT`}
```

Comment:
Nice! I like the consistency between this component and the other input adjacent components.